### PR TITLE
[DOCFIX] Portuguese site modification

### PR DESCRIPTION
**Pull request by caadecarvalho**
Two minor modifications, one to fit the site categories correctly and the other one is to fix 'Current Features' frame.

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/_config.yml [hunk 1]
```diff
@@ -47,11 +47,11 @@ categories:
   Frameworks:
     en: Frameworks
     cn: 计算框架
-    pt: Estruturas
+    pt: Frameworks
```
Comment (aaudiber) [inline docs/_config.yml]: Untranslating this doesn't save any letters, it's probably better to keep it in Portuguese

Comment (caadecarvalho) [inline docs/_config.yml]: Yes you are right but as there is no "good" direct translation to Portuguese for 'Frameworks', it is best to leave it in English

Comment (aaudiber) [inline docs/_config.yml]: I see, sgtm

**Comment by aaudiber** [comment]
Thanks for these fixes!

+1 LGTM

**Pull request merged**